src/api.rs

```diff
@@ -179,12 +179,18 @@ pub enum WatchEvent {
     },
 }
 
-#[derive(Debug, Clone)]
+#[derive(Debug, Clone, PartialEq)]
 pub enum User {
     None,
     Id(i32)
```

Comment:
Some documentation would be nice. Looking at this, I have absolutely no clue what a `User` is.